docs/contents/components/forms/fieldset/index.ja.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: Fieldset
+description: "`Fieldset`は、フィールドセット要素に、レジェンド、ヘルパーメッセージ、エラーメッセージなどをグループ化するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/form-control"
```

Comment:
Is it included in `@yamada-ui/form-control`?

Reply:
@108yen 

Yes, `Fieldset` exists in the package `@yamada-ui/form-control`.